webhook/env.go

```diff
@@ -37,3 +40,17 @@ func PortFromEnv(defaultPort int) int {
 	}
 	return port
 }
+
+func NameFromEnv() string {
+	if webhook := os.Getenv(webhookNameEnv); webhook != "" {
+		return webhook
+	}
+
+	panic(fmt.Sprintf(`The environment variable %q is not set
```

Comment:
Why would this be required? Can't it be sensibly defaulted?

Reply:
As it's being done for the `PortFromEnv` ? or you mean just set a hardcoded default value if anything is not set in env?

Reply:
@vagababov I think requiring it is better than defaulting it.  If we default it for two components in a namespace, then they will end up with the same lease names and have a bad time.  I helped the Tekton folks debug this with pipelines and triggers collocated recently.

We should probably change the panic message to indicate that this should be unique for the webhooks in a namespace.